data/initial_questions.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "initial_questions",
+  "data": [
+    {
+      "index": 0,
+      "image": "  _____     \n /     \\    \n|  O  O |   \n|   |   |   \n|   _   |   \n \\_____/    \n     |      \n     |      \n  ____|____ \n |         | \n |         | \n  ___________",
+      "question": "The alarm rings at 6 AM. What’s your reaction?",
+      "options": [
+        { "text": "Snooze it for “just 5 more minutes.”", "points": { "money": 0, "health": -5, "happiness": 0 } },
```

Comment:
Check the file, https://github.com/TheShiveshNetwork/kernel-panic/blob/main/data/studyabroad.json
The points field are health, happiness and wealth. Not money!